docs/src/languages/php/fpm.md

```diff
@@ -3,70 +3,107 @@ title: "PHP-FPM sizing"
+For example, if your application consumes on average `110 MB` of memory for a request, you can use:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 runtime:
     sizing_hints:
         request_memory: 110
 ```
 
-The `request_memory` has a lower limit of 10 MB while `reserved_memory` has a lower limit of 70 MB.
-Values lower than those are replaced with those minimums.
+Note that the minimum value for the `request_memory` key is 10 MB
+and the minimum value for the `reserved_memory` key is 70 MB.
+If you set lower values, they are automatically overridden with those minimums.
+
+For information on how to estimate the best `request_memory` value depending on your needs, 
+see how to [measure PHP worker memory usage](./fpm.md#measure-php-worker-memory-usage). 
 
-You can check the maximum number of PHP-FPM workers by opening an [SSH session](/development/ssh/_index.md) and running following command (example for PHP 7.x):
+To check the maximum number of PHP-FPM workers available to your site,
+open an [SSH session](/development/ssh/_index.md) and run the following command:
 
 ```bash
 grep -e '^pm.max_children' /etc/php/*/fpm/php-fpm.conf
+```
+
+You get output similar to the following,
+where workers are referred to as `children`:
+
+```bash
 pm.max_children = 2
 ```
 
-## Measuring PHP worker memory usage
+## Measure PHP worker memory usage
 
-To see how much memory your PHP worker processes are using, you can open an [SSH session](/development/ssh/_index.md) and look at the PHP access log:
+To see how much memory your PHP worker processes use, 
+open an [SSH session](/development/ssh/_index.md) 
+and look at the PHP access log:
 
 ```bash
 less /var/log/php.access.log
 ```
 
-In the fifth column, you'll see the peak memory usage that occurred while each request was handled. The peak usage will probably vary between requests, but to avoid the severe performance costs that come from swapping, your size hint should be somewhere between the average and worst case memory usages that you observe.
+In the fifth column of the output, 
+you can see the peak memory usage that occurred when each request was handled. 
+The optimal request memory is somewhere between the average and worst case memory usages listed.
 
-A good way to determine an optimal request memory is with the following command:
+To help determine what the optimal request memory is, run the following command:
 
 ```bash
 tail -n5000 /var/log/php.access.log | awk '{print $6}' | sort -n | uniq -c
 ```
 
-This will print out a table of how many requests used how much memory, in KB, for the last 5000 requests that reached PHP-FPM.
-(On an especially busy site you may need to increase that number).
-As an example, consider the following output:
+Note that the above command takes into account the last 5000 requests that reached PHP-FPM.
+You can adjust this number depending on the amount of traffic on your site.
```

Comment:
:+1: :hammer_and_wrench: This is more clear that the original, for sure. We do try to avoid directional language like `above`.
```suggestion
This command takes into account the last 5,000 requests that reached PHP-FPM.
You can adjust this number depending on the amount of traffic on your site.
```